zk/README.md

```diff
@@ -18,18 +18,6 @@ The ZooKeeper check is included in the [Datadog Agent][3] package, so you don't
 
 As of version 3.5, ZooKeeper has a `4lw.commands.whitelist` parameter (see [ZooKeeper documentation][7]) that whitelists [four letter word commands][8]. By default, only `srvr` is whitelisted. Add `stat` and `mntr` to the whitelist, as the integration is based on these commands.
 
-<!-- xxx tabs xxx -->
-<!-- xxx tab "Host" xxx -->
-
-#### Host
-
-To configure this check for an Agent running on a host:
-
-1. Edit the `zk.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4] to start collecting your ZooKeeper [metrics](#metric-collection) and [logs](#log-collection).
-   See the [sample zk.d/conf.yaml][5] for all available configuration options.
-
-2. [Restart the Agent][6].
-
 #### Enabling SSL
 _Available for Agent versions >=7.24.0 and ZooKeeper integration versions >=2.9.0. [See Changelog][14]._
```

Comment:
Should we remove this note now that the check is released and we're on 7.25

Reply:
Would it still be useful for debugging purposes?

Reply:
Our changelog is pretty clear IMO https://github.com/DataDog/integrations-core/blob/master/zk/CHANGELOG.md#290--2020-11-18

Reply:
But we can wait till 7.25 is released

Reply:
Gotcha, I've updated the PR that added SSL to ZooKeeper with the note about versioning. In that case I can remove this from the `README`.